### DISCUSSION TITLE
Improving Dev Cheats for a More Inclusive Community

### DISCUSSION THREAD

**Discussion by excalith** [excalith/dev-cheats#16] [Ideas]
Hello fellow open-sourcerers!

Before we start building our roadmap, we would like to focus on improving our documentation database to ensure that more people can use Dev Cheats. This will enable us to receive more feature requests, which will allow us to build a roadmap that meets the needs of the community, rather than just our own preferences.

We encourage all members of the community to provide feedback, suggestions, and ideas. Your input is critical to the success of this project, and we're committed to listening to your needs. Please feel free to share your feedback on this discussion or reach out to us directly regarding the roadmap.

Thank you for your interest in Dev Cheats,
Cheers!

**Comment by issafy** [comment]
Hey hi. This whole project is a really great initiative. The one app close to this I can remember and that is free is Zeal and it's user interface could use a big overhaul. 
I would really like to help this project grow anyway I can.🔥🔥🔥

**Reply by excalith** [reply]
Hey @SkywalkerD99 thanks for your interest, I really appreciate it! I am hoping to turn this project into something everyone can contribute and decide how it should evolve. Cheers!

**Reply by issafy** [reply]
Cheers
